solutions/USACO dec2020-cowntagion.mdx

```diff
@@ -0,0 +1,111 @@
+---
+id: usaco-1062
+source: USACO Silver 2020 December
+title: Cowntagion
+author: Kevin Sheng
+---
+
+[Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class Cowntagion {
+    public static void main(String[] args) throws IOException {
+        BufferedReader read = new BufferedReader(new InputStreamReader(System.in));
+        int farmNum  = Integer.parseInt(read.readLine());
+        ArrayList<Integer>[] neighbors = new ArrayList[farmNum];
```

Comment:
I think in java it's actually better to use the list interface, like:
`List<T>[] ... = new ArrayList[...];`
instead of creating an arrlist object. Same with arraydeque vs. deque interface. 

Can @nchn27 or some other java expert verify?

Reply:
Yea, use the List interface unless you need specific functions that aren't included in the interface.